ext/mysqli/mysqli.stub.php

```diff
@@ -547,225 +547,225 @@ function mysqli_connect_errno(): int {}
 
-function mysqli_stmt_free_result(mysqli_stmt $mysql_stmt): void {}
+function mysqli_stmt_free_result(mysqli_stmt $stmt): void {}
 
 #if defined(MYSQLI_USE_MYSQLND)
-function mysqli_stmt_get_result(mysqli_stmt $mysql_stmt): mysqli_result|false {}
+function mysqli_stmt_get_result(mysqli_stmt $stmt): mysqli_result|false {}
 #endif
 
-function mysqli_stmt_get_warnings(mysqli_stmt $mysql_stmt): mysqli_warning|false {}
+function mysqli_stmt_get_warnings(mysqli_stmt $stmt): mysqli_warning|false {}
 
-function mysqli_stmt_init(mysqli $mysql_link): mysqli_stmt|false {}
+function mysqli_stmt_init(mysqli $connection): mysqli_stmt|false {}
 
-function mysqli_stmt_insert_id(mysqli_stmt $mysql_stmt): int|string {}
+function mysqli_stmt_insert_id(mysqli_stmt $stmt): int|string {}
 
 #if defined(MYSQLI_USE_MYSQLND)
-function mysqli_stmt_more_results(mysqli_stmt $mysql_stmt): bool {}
+function mysqli_stmt_more_results(mysqli_stmt $stmt): bool {}
 
-function mysqli_stmt_next_result(mysqli_stmt $mysql_stmt): bool {}
+function mysqli_stmt_next_result(mysqli_stmt $stmt): bool {}
 #endif
 
-function mysqli_stmt_num_rows(mysqli_stmt $mysql_stmt): int|string {}
+function mysqli_stmt_num_rows(mysqli_stmt $stmt): int|string {}
 
-function mysqli_stmt_param_count(mysqli_stmt $mysql_stmt): int {}
+function mysqli_stmt_param_count(mysqli_stmt $stmt): int {}
 
-function mysqli_stmt_prepare(mysqli_stmt $mysql_stmt, string $query): bool {}
+function mysqli_stmt_prepare(mysqli_stmt $stmt, string $query): bool {}
 
-function mysqli_stmt_reset(mysqli_stmt $mysql_stmt): bool {}
+function mysqli_stmt_reset(mysqli_stmt $stmt): bool {}
 
-function mysqli_stmt_result_metadata(mysqli_stmt $mysql_stmt): mysqli_result|false {}
+function mysqli_stmt_result_metadata(mysqli_stmt $stmt): mysqli_result|false {}
 
-function mysqli_stmt_send_long_data(mysqli_stmt $mysql_stmt, int $param_nr, string $data): bool {}
+function mysqli_stmt_send_long_data(mysqli_stmt $stmt, int $index, string $data): bool {}
 
-function mysqli_stmt_store_result(mysqli_stmt $mysql_stmt): bool {}
+function mysqli_stmt_store_result(mysqli_stmt $stmt): bool {}
 
-function mysqli_stmt_sqlstate(mysqli_stmt $mysql_stmt): ?string {}
+function mysqli_stmt_sqlstate(mysqli_stmt $stmt): ?string {}
 
-function mysqli_sqlstate(mysqli $mysqli_link): ?string {}
+function mysqli_sqlstate(mysqli $connection): ?string {}
 
 function mysqli_ssl_set(
-    mysqli $mysql_link,
+    mysqli $connection,
     string $key,
-    string $cert,
+    string $certificate,
     string $certificate_authority,
     string $certificate_authority_path,
-    string $cipher
+    string $cipher_algorithms
 ): bool {}
 
-function mysqli_stat(mysqli $mysql_link): string|false {}
+function mysqli_stat(mysqli $connection): string|false {}
 
-function mysqli_store_result(mysqli $mysql_link, int $flags = 0): mysqli_result|false {}
+function mysqli_store_result(mysqli $connection, int $flags = 0): mysqli_result|false {}
```

Comment:
The use of `$flags` suggests that it is a bitmask for setting options, while it is not. I think `$mode` would be more appropriate here.

https://www.php.net/manual/en/mysqli.store-result.php